### DISCUSSION TITLE
Adding a createOptionUsing section in Forms/Select documentation

### DISCUSSION THREAD

**Discussion by Carnicero90** [filamentphp/filament#11533] [Ideas]
@danharrin I think it's worth adding a dedicated section explaining the `createOptionUsing` method, since the return value wasn't that clear to me, at a first sight (I was returning the whole model, instead of the id).

If agreeable, I am willing to implement it.

**Comment by danharrin** [comment]
Sounds good

**Reply by Carnicero90** [reply]
okok done in #11535 (hopefully)
